config/config.go

```diff
@@ -265,6 +265,7 @@ type Config struct {
 	OauthTokenExpire                  int32                                 `json:"oauth_token_expire"`
 	OauthTokenExpiredRetainPeriod     int32                                 `json:"oauth_token_expired_retain_period"`
 	OauthRedirectUriSeparator         string                                `json:"oauth_redirect_uri_separator"`
+	PaginationItemsPerPage            int                                   `json:"pagination_items_per_page"`
```

Comment:
if it is a global param for all paginated data  - name is OK, if it is specific to oauth tokens pagination I'd prefer something like `oauth_tokens_per_page`

Reply:
I didn't find anything related to pagination so I thought a ___global config value___ would be it. Fine to change it to the specified name if OAuth tokens are the only place pagination will be needed.